### DISCUSSION TITLE
Suggestion: Add labels for `@remotion` packages for use with issues and PRs

### DISCUSSION THREAD

**Discussion by with-heart** [remotion-dev/remotion#2393] [Ideas]
It would be helpful and healthy for the [`remotion-dev/remotion` repo](https://github.com/remotion-dev/remotion) if labels were created for each `@remotion` package and added to relevant issues and PRs. I know this adds a bit of a maintenance burden but I believe the benefits outweigh the costs:

- _users_ can find all issues relevant to a specific package
- _contributors_ can find PRs for a specific package, enabling them to understand how the package was built
- _anyone_ can follow the history of a specific package as told by its issues and PRs

**Comment by JonnyBurger** [comment]
Thanks for the suggestion!
We value if it is easy to file issues without much requirements (often just dumping observations while working on something else), but I can try to include the full package name in the description of the issue so that at least you use search in the future.

**Reply by with-heart** [reply]
I think that's a fair compromise ❤️
